README.md

```diff
@@ -62,6 +66,46 @@ $ npm run start:dev
 
 ```
 
-### Automated monitoring
+## Instructions for anchors
+
+### Interactive flow instrumentation
+
+The interactive flow of SEP24 is custom for each anchor which makes it difficult
+to automate. In order to help the automated tests complete your interactive
+flow, any form fields need to be annotated with expected values.
+
+There are two things that need to be done: provide valid values for each field,
+and identify which button should be pressed to continue.
+
+#### Providing field values
+
+Any form field that is required should have a `test-value` attribute which is
+set to a valid value for that field. For example an email field would look like
+`<input type='text' id='email_address' test-value='dummyaddress2342@gmail.com' />`.
+The number can be randomized at render time to avoid reuse.
```

Comment:
I'm not sure this last point is necessary, is there a reason an anchor would want to change the dummy values?

Reply:
It can cause problems if we test with new stellar addresses but existing email addresses